src/components/SummaryCard/_mixins.scss

```diff
@@ -4,17 +4,22 @@
 /// @copyright IBM Security 2019
 ////
 
-@import '../../globals/index';
+@import 'carbon-components/scss/globals/scss/helper-mixins';
+@import 'carbon-components/scss/globals/scss/vars';
 
-/// Use until Carbon fixes `text-overflow` helper mixin.
-/// @access private
-@mixin summary-card--ellipsis {
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-}
```

Comment:
Hey do you mind keeping this & restoring it's usage? I don't believe the `text-overflow` helper mixin has been changed. Specifically this line: https://github.com/carbon-design-system/carbon/blob/master/packages/components/src/globals/scss/_helper-mixins.scss#L34

Reply:
I wonder if this is related to https://github.com/carbon-design-system/ibm-security/pull/228#issuecomment-562574234 and the disparity between testing environments - I'm happy to restore once we can confirm the issue is still occurring.

Reply:
Yeah sounds good -- I wish I more accurately remembered what he issue was that I was trying to fix. 😅 I'll look into it & post if I re-discover what the original problem was.

Reply:
Actually disregard -- I don't believe the issue is occurring anymore, and actually based on my local testing just now I think `overflow-x: hidden` is very vital.